server/src/routes/roomsRoutes.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import express from 'express';
+import { CommunicationIdentityClient } from '@azure/communication-identity';
+import { RoomsClient } from '@azure/communication-rooms';
+import { createRoom, joinRoom } from '../controllers/roomsController';
+
+export const roomsRouter = (identityClient: CommunicationIdentityClient, roomsClient: RoomsClient): express.Router => {
+  // Initialize router
+  const router = express.Router();
+
+  router.post('/', createRoom(identityClient, roomsClient));
+  router.get('/join', joinRoom(identityClient, roomsClient));
```

Comment:
I'm not sure if we should call this 'join'. To me the word join sounds like the participant is being added to the room. But this API is just getting token. Maybe we should call it 'token'? Also I would suggest we use POST because there is a resource (token) being created

Reply:
That is a good point. I actually had similar question but different reason. I will speak with @ralphgabrinao tomorrow and leave the conclusion here.

Reply:
Agreed, I like `/api/rooms/token`

Reply:
@ralphgabrinao -- Since we change this from `GET` to `POST` method, should we send the `userId` and `roomId` as body payload instead of query string?

Reply:
I think it will have to stay as query string because the idea is that users can join the room using a link: `/visit?userId=<myUserId>&roomId=<myRoomId>` 

If the userId and roomId are put in the body instead, it wouldn't be possible to join using a link like above.

Reply:
Yeah, putting in the body payload works. Then the e2e flow would be:

- User pastes this url in the browser: `myapp.com/visit?userId=<myUserId>&roomId=<myRoomId>`
- MeetingExperience component extracts the userId and roomId from query string
- MeetingExperience calls this endpoint and puts userId and roomId in the body payload to get token
- MeetingExperience renders CallComposite